src/util.c

```diff
@@ -37,6 +37,9 @@
 #define NI_BYTE_ARRAY_CHUNK	0xff	/* important: (2^n)-1 with all bits set */
 #define NI_BYTE_ARRAY_SIZE(len)	((len) | NI_BYTE_ARRAY_CHUNK)
 
+#define NI_BIT_SEPERATOR_FORMAT	"|"
+#define NI_BIT_SEPERATOR_PARSE	" ,|\t\n"
```

Comment:
Also for parsing, even it's the same?

Reply:
For parsing we allow almost all separators, see
* https://github.com/openSUSE/wicked/blob/master/src/dbus-xml.c#L563
* https://github.com/openSUSE/wicked/blob/master/src/dbus-xml.c#L600

But maybe we could do it for consistency.

Reply:
Yes, the parsing of both is the same.